### PR TITLE
Wildflyserver fips assessment

### LINKED ISSUE #29: Add eap operator to fips-assessments tasks
Add Ansible tasks that can create the following CR(s) and validate non-failed status: https://github.com/opdev/fips-assessments/blob/main/eap-operator

### PR DESCRIPTION
**Pull request by makon57**
Resolves #29

### CODE REVIEW

File: eap/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+- name: Setting Up Results Var
+  ansible.builtin.set_fact:
+    test_result: "pass"
```
Comment (bcrochet) [inline eap/tasks/main.yml:4]:
```suggestion
    assessment_result: "pass"
```

File: eap/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        - wildfly_info.resources[0].status is defined
+        - wildfly_info.resources[0].status.hosts is defined
+        - wildfly_info.resources[0].status.pods[0].state is defined
+        - wildfly_info.resources[0].status.pods[0].state == "ACTIVE"
+      retries: 3
+      delay: 10
+  rescue:
+    - name: Print when failed
+      ansible.builtin.debug:
+        msg: "WildFlyServer failed. Err: {{ ansible_failed_result }}"
```
Comment (bcrochet) [inline eap/tasks/main.yml:40]: You'll need to set assessment_result: "fail" in this block.

**Review by bcrochet — CHANGES_REQUESTED** [review]
Just need to use assessment_result to pass the result back, but otherwise looks good!

File: eap/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    - name: Register failed tests
+      ansible.builtin.set_fact:
+        assessment_result: "fail"
+# print pass or failed according to test_results
+- name: WildFlyServer Results
+  ansible.builtin.debug:
+    msg: "{% if test_result == 'pass' %} PASS {% else %} FAILED {% endif %}"
```
Comment (acmenezes) [inline eap/tasks/main.yml:47]: Hi @makon57 , here test result shows up as undefined for me. I guess you want assessment_result is that it? And you don't necessarily need to print the result on the screen.

**Review by acmenezes — CHANGES_REQUESTED** [review]

File: eap/tasks/main.yml (hunk 3 above)
Comment (bcrochet) [reply eap/tasks/main.yml:47]: Yeah, I missed this one. I concur with Alex. Keeping this is optional.

Comment (makon57) [reply eap/tasks/main.yml]: Changed it to assessments and left it as is.

**Review by bcrochet — COMMENTED** [review]
/lgtm

**Review by acmenezes — APPROVED** [review]
Running flawlessly! Thanks @makon57 !
/lgtm

**Pull request merged**